addon/initializers/register-let-helper.js

```diff
@@ -17,22 +13,6 @@ export default {
         options.template.yield(params);
       }
```

Comment:
For < 2.0 users, shouldn't there be an `else` with a warning that inline usage is not allowed?